packages/@okta/vuepress-site/docs/reference/user-query/index.md

```diff
@@ -0,0 +1,716 @@
+            }
+        }
+    },
+    {
+        "id": "00u3m5wrdPjJYUFb81d6",
+        "status": "LOCKED_OUT",
+        "created": "2020-10-22T15:15:30.000Z",
+        "activated": "2020-10-22T15:15:30.000Z",
+        "statusChanged": "2020-10-22T16:09:13.000Z",
+        "lastLogin": null,
+        "lastUpdated": "2020-10-22T16:09:13.000Z",
+        "passwordChanged": "2020-10-22T15:15:30.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "John",
+            "lastName": "Mclean",
+            "mobilePhone": null,
+            "secondEmail": null,
+            "login": "johnmclean@example.com",
+            "email": "johnmclean@example.com"
+        },
+        "credentials": {
+            "password": {},
+            "emails": [
+                {
+                    "value": "johnmclean@example.com",
+                    "status": "VERIFIED",
+                    "type": "PRIMARY"
+                }
+            ],
+            "provider": {
+                "type": "IMPORT",
+                "name": "IMPORT"
+            }
+        },
+        "_links": {
+            "self": {
+                "href": "https://{yourOktaDomain}/api/v1/users/00u3m5wrdPjJYUFb81d6"
+            }
+        }
+    },
+    {
+        "id": "00u3ojdzgjrAhuYGg1d7",
+        "status": "ACTIVE",
+        "created": "2022-05-13T20:06:20.000Z",
+        "activated": "2022-05-13T20:06:22.000Z",
+        "statusChanged": "2022-05-13T20:06:22.000Z",
+        "lastLogin": null,
+        "lastUpdated": "2022-05-13T20:06:22.000Z",
+        "passwordChanged": "2022-05-13T20:06:21.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "Jack",
+            "lastName": "Phillips",
+            "mobilePhone": null,
+            "secondEmail": null,
+            "login": "john.j.phillips@example.com",
+            "email": "john.j.phillips@example.com"
+        },
+        "credentials": {
+            "password": {},
+            "emails": [
+                {
+                    "value": "john.j.phillips@example.com",
+                    "status": "VERIFIED",
+                    "type": "PRIMARY"
+                }
+            ],
+            "provider": {
+                "type": "OKTA",
+                "name": "OKTA"
+            }
+        },
+        "_links": {
+            "self": {
+                "href": "https://{yourOktaDomain}/api/v1/users/00u3ojdzgjrAhuYGg1d7"
+            }
+        }
+    }
+```
+
+See [Find Users](/docs/reference/api/users/#find-users) in the Users API reference.
+
+## List all users
+
+The basic list-all-users command returns all users, that don't have a `DEPROVISIONED` status, to the default maximum limit of 200 records, if not specified. For this simple use case, the list-all query performance is superior to other search options.
```

Comment:
This feels redundant ("list all users command returns all users") and the hyphens feel clunky.  Do we really need to give this its own name, or could we say, "If a query does not include any of the parameters `search`, `filter`, or `q`, it returns all users (except those with `DEPROVISIONED` status), with a default limit of 200 records."

Reply:
Thanks. Reads much better.